content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: XXXX'
+date: 2021-04-08
+slug: kubernetes-1-21-release-announcement
+---
+
+**Authors:** [Kubernetes 1.21 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release_team.md)
```

Comment:
BTW, https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release_team.md is a 404

Reply:
https://github.com/kubernetes/sig-release/tree/master/releases/release-1.21/release-team.md should be the correct path. Will modify it, thank you for the catch!

Reply:
This has now been modified in the latest commit :)